src/extension.ts

```diff
@@ -37,17 +37,17 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
         activateContext.telemetry.properties.isActivationEvent = 'true';
         activateContext.telemetry.measurements.mainFileLoad = (perfStats.loadEndTime - perfStats.loadStartTime) / 1000;
 
-        registerCommand('azureVirtualMachines.createVirtualMachine', createVirtualMachine);
-        registerCommand('azureVirtualMachines.createVirtualMachineAdvanced', createVirtualMachineAdvanced);
-        registerCommand('azureVirtualMachines.startVirtualMachine', startVirtualMachine);
-        registerCommand('azureVirtualMachines.restartVirtualMachine', restartVirtualMachine);
-        registerCommand('azureVirtualMachines.stopVirtualMachine', stopVirtualMachine);
-        registerCommand('azureVirtualMachines.addSshKey', addSshKey);
-        registerCommand('azureVirtualMachines.deleteVirtualMachine', deleteVirtualMachine);
-        registerCommand('azureVirtualMachines.copyIpAddress', copyIpAddress);
-        registerCommand('azureVirtualMachines.openInRemoteSsh', openInRemoteSsh);
-        registerCommand('azureVirtualMachines.showOutputChannel', () => { ext.outputChannel.show(); });
-        registerCommand('azureVirtualMachines.showRemoteSshExtension', () => { void vscode.commands.executeCommand('extension.open', remoteSshExtensionId); });
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.createVirtualMachine', createVirtualMachine);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.createVirtualMachineAdvanced', createVirtualMachineAdvanced);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.startVirtualMachine', startVirtualMachine);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.restartVirtualMachine', restartVirtualMachine);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.stopVirtualMachine', stopVirtualMachine);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.addSshKey', addSshKey);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.deleteVirtualMachine', deleteVirtualMachine);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.copyIpAddress', copyIpAddress);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.openInRemoteSsh', openInRemoteSsh);
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.showOutputChannel', () => { ext.outputChannel.show(); });
+        registerCommandWithTreeNodeUnwrapping('azureVirtualMachines.showRemoteSshExtension', () => { void vscode.commands.executeCommand('extension.open', remoteSshExtensionId); });
```

Comment:
These two commands aren't invoked from the tree. So they can still use `registerCommand`.